src/main/java/com/google/ehub/servlets/LoginServlet.java

```diff
@@ -34,28 +31,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
+    JsonObject loginJson = new JsonObject();
 
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      String urlToRedirectToAfterUserLogsOut = "/?authuser=0";
-      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      String logoutUrl = userService.createLogoutURL("/?authuser=0");
+      loginJson.addProperty("LogoutURL", logoutUrl);
+      loginJson.addProperty("LoginURL", "");
```

Comment:
Lets add some constants for the hard-coded strings since they are used in multiple places:
```
private static final String LOGOUT_URL_KEY = LogoutURL;
private static final String LOGIN_URL_KEY = LoginURL;
```
Then you can use the constants here and below.
```suggestion
      loginJson.addProperty(LOGOUT_URL_KEY, logoutUrl);
      loginJson.addProperty(LOGIN_URL_KEY, "");
```